groovy/postInit/chains/PollutionChain.groovy

```diff
@@ -66,4 +79,78 @@ SMOKE_STACK.recipeBuilder()
 SMOKE_STACK.recipeBuilder()
         .fluidInputs(fluid('sour_gas') * 500)
         .duration(20)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('water') * 100)
+        .inputs(metaitem('dustTinyAsh'))
+        .fluidOutputs(fluid('ash_slurry') * 100)
+        .duration(5)
+        .EUt(7)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('water') * 100)
+        .inputs(metaitem('dustTinyDarkAsh'))
+        .fluidOutputs(fluid('ash_slurry') * 100)
+        .duration(5)
+        .EUt(7)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('water') * 1000)
+        .inputs(metaitem('dustAsh'))
+        .fluidOutputs(fluid('ash_slurry') * 1000)
+        .duration(40)
+        .EUt(7)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('water') * 1000)
+        .inputs(metaitem('dustDarkAsh'))
+        .fluidOutputs(fluid('ash_slurry') * 1000)
+        .duration(40)
+        .EUt(7)
+        .buildAndRegister()
+
+DUMPER.recipeBuilder()
+        .fluidInputs(fluid('ash_slurry') * 1000)
+        .duration(20)
+        .buildAndRegister()
+
+DUMPER.recipeBuilder()
+        .fluidInputs(fluid('red_mud') * 1000)
+        .duration(20)
+        .buildAndRegister()
+
+DUMPER.recipeBuilder()
+        .fluidInputs(fluid('concentrated_red_mud') * 1000)
+        .duration(20)
+        .buildAndRegister()
+
+SIFTER.recipeBuilder()
+        .fluidInputs(fluid('flue_gas') * 16000)
+        .inputs(metaitem('dustCalcite') * 5)
+        .fluidOutputs(fluid('desulfurized_flue_gas') * 16000)
+        .outputs(metaitem('dustCalciumSulfite') * 5)
+        .duration(160)
+        .EUt(30)
+        .buildAndRegister()
+
+ROASTER.recipeBuilder()
```

Comment:
These 2 reactions are impossible. Instead, only the following can be achieved.

CaSO3 -roaster-> CaO + SO2

CaSO3 + 1/2 O2 --> CaSO4